src/components/LogPane/index.tsx

```diff
@@ -32,10 +32,11 @@ const LogPane: React.FC<LogPaneProps> = ({
         <List
           // If wrap is false, We want the list to scroll horizontally.
```

Comment:
nit. I think you can remove this comment since we are no longer conditionally applying the `containerStyle`